std/math.d

```diff
@@ -703,12 +703,10 @@ deprecated
  */
 
 real cos(real x) @safe pure nothrow @nogc { pragma(inline, true); return core.math.cos(x); }
-//FIXME
 ///ditto
-double cos(double x) @safe pure nothrow @nogc { return cos(cast(real) x); }
-//FIXME
+double cos(double x) @safe pure nothrow @nogc { pragma(inline, true); return core.math.cos(x); }
 ///ditto
-float cos(float x) @safe pure nothrow @nogc { return cos(cast(real) x); }
+float cos(float x) @safe pure nothrow @nogc { pragma(inline, true); return core.math.cos(x); }
```

Comment:
You haven't actually changed anything yet. `core.math.cos(x)` is defined as

```d
real cos(real x) @safe pure nothrow;
```

So the argument is still being converted to `real`. Same throughout.

Reply:
see dlang/druntime#2135

Reply:
Is there a reason `public import core.math : cos` wouldn't be better?

Reply:
Good point. I'll wait for the DMD/Druntime prs first though, since this needs them.